omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -301,6 +301,16 @@
             });
 
             jq(document).ready(function () {
+
+                /* Prompt if client is on TB Rx or completed it */
+                const lastTBStatusValue = jq("#lastTBStatus").html().split('#');
```

Comment:
I couldnt get a covenient way to get the conceptId of lastTBStatus...I tried <span style="display: none" id="lastTBStatus"><lookup expression="fn.latestObs('90216').valueCoded.concetId"/></span> but it doesnt work.
This is why I manipulated the output

Reply:
@reagan-meant use getValue('fieldname-id')